### ISSUE TITLE
Produce fresh OSCAL profiles representing baselines described in NIST SP800-53B

### ISSUE THREAD

**Issue by wendellpiez** [usnistgov/oscal-content#30]
At time of writing, a final release of NIST 800-53B, describing control baselines, has been or is being made available.

For the FPD we produced four OSCAL profiles representing baselines HIGH, MODERATE, LOW and PRIVACY.

We should refresh this work based on the latest release, and update the published profiles accordingly.

**Comment by wendellpiez** [comment]
Now in a PR, see https://github.com/usnistgov/oscal-content/pull/32

**Comment by wendellpiez** [comment]
## Update Nov 5 2020

The PR contains profiles, but not resolved profiles. To produce these we need a stable final catalog, so this Issue is dependent on #25 finalizing the OSCAL for SP800-53rev5.

Potentially also an Issue relating to the design of the PRIVACY baseline, could perturb this. Discussion needed.

**Comment by david-waltermire-nist** [comment]
@wendellpiez Can you run `build/ci-cd/copy-and-convert-content.sh --resolve-profiles` and spot check that the resolved catalogs make sense? 

This issue is related to usnistgov/OSCAL#771

**Comment by wendellpiez** [comment]
Running on my system I can confirm the bug described in usnistgov/OSCAL/#771 - `param/select` is failing to propagate. This needs to be debugged through unit testing. I am making a development branch in the OSCAL repo for that purpose and for the purposes of #35.

Suggest consolidating profile resolution work items on an Issue? Punchlist:

- [ ] Resolve #771 so `param/select` is propagated; with unit tests
- [ ] Stress test profile resolution of PRIVACY baselines (#35) against an appropriate scenario (either 'uncle/nephew' or 'cousins' profile imports); extend functionality with new unit tests as appropriate
- [ ] Regression test everything

**Comment by wendellpiez** [comment]
We have PR https://github.com/usnistgov/OSCAL/pull/788 to address the first issue.

A separate Issue can be carved off for further testing and extension of profile resolution to support the FISMA use case (and related), once those are clarified.

As of today Nov 17 we met with the FISMA team and the situation wrt the PRIVACY baselines is indeed interesting and potentially complex, as within NIST SP800-53B, the relation between the PM controls and the subset of PM controls addressing privacy, is complicated by the fact that in general the PM controls are not managed at the system level (ordinarily they would be managed at the level of the enterprise), and thus correspond to no *system-* level baseline or assessment.

While the profile model can readily group and subset controls for purposes of "mix and match" this does raise more operational questions regarding the representation of logical dependencies between controls. Accordingly, even while a number of provisional designs for OSCAL profiles describing the 800-53B baselines are conceivable, there may be no "right" answer only a best one.

The current PR https://github.com/usnistgov/oscal-content/pull/32 has three composite baselines including the PRIVACY controls. Perhaps this Issue could be closed (with some questions deferred till later) if we added a `readme` describing the situation with the composite catalogs and issues potentially raised while processing them.

Question: if the PM controls are not included in the security baselines, should they be collected into their own profile model for inclusion into a profile that needs them? What is the subset of PM controls that belong in such a profile?

**Comment by wendellpiez** [comment]
**Update Dec 3 2020**

The security baselines in PR oscal-content#32 look good.

Once they are merged, we can close this issue, as we have #35 to track the Privacy baselines.

**Comment by david-waltermire-nist** [comment]
This was addressed with the merge or PR #32.
